README.md

```diff
@@ -9,7 +9,7 @@ This information can be updated quite often, but is all peer reviewed. If you se
 * [Scrum Process](process/process.md)
 * [Sprint Development](process/sprints.md)
 * [Feature Toggles](process/feature-toggles.md)
-
+* [Deployment](deployment/Rules-Publish-TST-To-PRD.md)
```

Comment:
This link should go to a page that houses all documentation for deployments. Then, on that page, we link to your publish for rules document. This will allow us to expand it in the future. For now, this might be fine though.